templates/blog/desktop.html

```diff
@@ -5,19 +5,15 @@
 {% block meta_description %}All you need to know about the fast, free and user-friendly Ubuntu desktop operating system.{% endblock %}
 
 {% block content %}
-
-  <section class="p-strip--image is-dark p-strip--blog-suru" style="background-position: center center; background-image:url('https://assets.ubuntu.com/v1/bf5e5af8-desktop-overview-hero.jpg')">
-    <div class="row u-equal-height u-vertically-center">
-      <div class="col-6 p-card--suru">
-        <h1 class="p-heading--one">Desktop</h1>
-        <h5 style="line-height: 1.75rem;">
-          All you need to know about the fast, free and user&#8209;friendly Ubuntu desktop operating system.
-        </h5>
-        <p>
-          <a class="p-link--external p-link--inverted" href="https://www.ubuntu.com/desktop">Learn more about Desktop</a>
-        </p>
-      </div>
-      <div id="rtp-banner" class="rtp-banner"></div>
+  <section class="p-strip--suru-blog-header">
+    <div class="u-fixed-width">
+      <h1>
+        Desktop
+      </h1>
+      <h2 class="p-heading--four">All you need to know about the fast, free and user <br class="u-hide u-show--large">friendly Ubuntu desktop operating system.</h2>
+      <p>
+        <a class="p-link--external p-link--inverted" href="https://www.ubuntu.com/desktop">Learn more about Desktop</a>
+      </p>
```

Comment:
Leave this in place:
```suggestion
      </p>
      <div id="rtp-banner" class="rtp-banner"></div> 
```